cg-producer/pycg_producer/producer.py

```diff
@@ -237,27 +238,17 @@ def _generate_callgraph(self, package_path):
             '--output', self.out_file.as_posix()
         ] + files_list
 
-        timing = [
-            "/usr/bin/time",
-            "-f", "secs=%e\nmem=%M"
-        ]
+        result = self._execute_with_benchmark(cmd)
 
-        try:
-            out, err = self._execute(timing + cmd)
-        except Exception as e:
-            self._format_error('generation', str(e))
+        if result["process"]["stderr_data"]:
+            self._format_error('generation', result["process"]["stderr_data"].strip())
             raise CallGraphGeneratorError()
 
         if not self.out_file.exists():
-            self._format_error('generation', err.decode('utf-8'))
+            self._format_error('generation', result["process"]["stderr_data"].strip())
             raise CallGraphGeneratorError()
-
-        for l in err.decode('utf-8').splitlines():
-            if l.strip().startswith("secs"):
-                self.elapsed = float(l.split("=")[-1].strip())
-            if l.strip().startswith("mem"):
-                self.max_rss = int(l.split("=")[-1].strip())
-
+        self.elapsed = result["process"]["execution_time"]
+        self.max_rss = round(result["memory"]["max"]/1000)
         return self.out_file
 
     def _get_python_files(self, package):
```

Comment:
the `resolve()` here causes an issue with Windows paths (even though it shouldn't...) so I propse to remove it. It doesn't seem to be needed either, everything works without.

Reply:
Sorry, I did not quite catch this, where the `resolve()` method is called?

Reply:
Removed on 1ac65af